posix.mak

```diff
@@ -277,7 +277,14 @@ verbatim : $(addprefix $(DOC_OUTPUT_DIR)/, $(addsuffix .verbatim,$(PAGES_ROOT)))
 
 kindle : ${DOC_OUTPUT_DIR}/dlangspec.mobi
 
+ifeq (1,$(DIFFABLE))
+# Remove /dlangspec.{tex,html} once DAutoTest has been updated
+# https://github.com/dlang/dlang.org/pull/1969
+pdf : ${DOC_OUTPUT_DIR}/dlangspec.pdf ${DOC_OUTPUT_DIR}/dlangspec.tex ${DOC_OUTPUT_DIR}/dlangspec.html \
+		dlangspec.tex dlangspec.html
```

Comment:
At the moment it has to build `dlangspec.{tex,html}` by default too as otherwise DAutotest will complain that it can't copy non-existent files.

Reply:
Sorry, this is not at all what I meant.

As I wrote in a previous comment, I think the best way would be a new target which copies the intermediaries to the output directory.

This way is more complicated than necessary, and is causing duplicated files to be copied.

Reply:
This is also technically incorrect because `dlangspec.html` does not participate in building the PDF, only the eBook.

Reply:
-> https://github.com/dlang/dlang.org/pull/1988

Reply:
Oh I got errors otherwise because DAutoTest was explicitly trying to copy `dlangspec.{tex,html}`
Anyhow I'm really sorry for breaking DAutoTest and thanks a lot for your quick fix. Really appreciated!

Reply:
Sorry if I miscommunicated. What I had in mind was you adding a target and me updating DAutoTest to use it. All's well now anyway.

Reply:
> Anyhow I'm really sorry for breaking DAutoTest

Auto-deploy is still broken... didn't look into that yet.

Reply:
I know why. Fix incoming.

Reply:
> Auto-deploy is still broken... didn't look into that yet.

-> https://github.com/dlang/dlang.org/pull/1990